platform/engineering/internal-tools.md

```diff
@@ -103,24 +101,24 @@ To create and use SSH keypairs on Windows, complete the following steps:
    
    `ls ~/.ssh`
    
-   You will see your public and private keys. Your public key will end in a `.pub` extension, such as `id_rsa_vetsgov.pub`, and your private key will have the same name minus the `.pub` extension. 
+   You will see your public and private keys. Your public key will end in a `.pub` extension, such as `id_rsa_vagov.pub`, and your private key will have the same name minus the `.pub` extension. 
 1. You can view the contents of your new public key by entering the following command: 
    
-   `cat ~/.ssh/id_rsa_vetsgov.pub`
+   `cat ~/.ssh/id_rsa_vagov.pub`
    
    You will need to copy the contents and provide them in a future step.
 1. Add this key to your SSH agent any time this terminal is restarted. 
    - Ensure your SSH agent is running by typing: `eval $(ssh-agent -s)`.
-   - Add your key to the agent by typing: `ssh-add ~/.ssh/id_rsa_vetsgov`.  
+   - Add your key to the agent by typing: `ssh-add ~/.ssh/id_rsa_vagov`.  
      **Note:** Use the private key here, not the `.pub` public key. 
    - Verify the key is added by typing: `ssh-add -l`. 
-     This command should give output showing your key's signature added to the running SSH agent like the illustration below.
-     
+     This command should give output showing your key's signature added to the running SSH agent like the illustration below:
      ```
      $ssh-add -l                                                                                           
      2048 SHA256:ShkbdHKQqDwgONLv8/1qiYlX20kX9IPp3uV56ATp3c8 
-     /home/user/.ssh/id_rsa_vetsgov (RSA)
+     /home/user/.ssh/id_rsa_vagov (RSA)
      ```
+   - Any time your terminal is restarted, you will want to ensure your key is added by running `ssh-add -l`. If your key signature is not returned, or you receive output referring to your agent, run `eval $(ssh-agent -s)` and `ssh-add ~/.ssh/id_rsa_vagov`.
```

Comment:
We can probably mention this for mac users if we don't elsewhere https://apple.stackexchange.com/questions/48502/how-can-i-permanently-add-my-ssh-private-key-to-keychain-so-it-is-automatically

Reply:
Good idea, will get it added.

Reply:
Added optional/additional options for automatically adding key to keychain/agent for Mac/Windows, respectively in https://github.com/department-of-veterans-affairs/va.gov-team/pull/11950/commits/4ef15766750f8efc1ebd042f18535d332c1d9a1f